components/style/values/specified/length.rs

```diff
@@ -222,14 +222,122 @@ impl CharacterWidth {
     }
 }
 
+/// Same as Gecko
+const ABSOLUTE_LENGTH_MAX: f64 = (1 << 30) as f64;
+const ABSOLUTE_LENGTH_MIN: f64 = - (1 << 30) as f64;
+
+/// Helper to convert a floating point length to application units
+fn to_au_round(length: CSSFloat, au_per_unit: CSSFloat) -> Au {
+    Au(
+        ((length as f64) * (au_per_unit as f64))
```

Comment:
Purely because it was in the sample in the issue, with the bounds we have there seems to be no need indeed.